Dockerfile

```diff
@@ -1,5 +1,18 @@
 FROM debian:wheezy
-RUN apt-get update -qq && apt-get install -qy python python-pip python-dev git && apt-get clean
+
+RUN apt-get update -qq
+
+# Compose dependencies
+RUN apt-get install -qqy python python-pip python-dev git
+
+# Test dependencies
+RUN apt-get install -qqy apt-transport-https ca-certificates curl lxc iptables
+RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.3.3 > /usr/local/bin/docker-1.3.3 && chmod +x /usr/local/bin/docker-1.3.3
+RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.4.1 > /usr/local/bin/docker-1.4.1 && chmod +x /usr/local/bin/docker-1.4.1
+RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.5.0 > /usr/local/bin/docker-1.5.0 && chmod +x /usr/local/bin/docker-1.5.0
+
+RUN apt-get clean
```

Comment:
It also won't actually do anything: https://github.com/docker/docker/blob/738bdec4908755761b27b7d15baeefb1a824f770/contrib/mkimage/debootstrap#L72-L95

The pattern we usually use is `RUN apt-get update && apt-get install -y ... && rm -rf /var/lib/apt/lists/*`.